hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -199,6 +259,16 @@ public Builder enable(boolean enable) {
       return this;
     }
 
+    public Builder withMetadataIndexBloomFilter(boolean enable) {
```

Comment:
Since this is MetadataConfig, it is implied that all of these are metadata related. So you may drop the word "Metadata" from these function names to shorten them.

Reply:
fixed